Makefile

```diff
@@ -26,9 +26,6 @@ cluster-prepare:
 	$(MAKE) -C third_party/cert-manager deploy
 	$(MAKE) -C third_party/registry deploy
 	$(MAKE) -C third_party/vault deploy
-
-.PHONY: cluster-prepare-wait
-cluster-prepare-wait:
```

Comment:
The split (cluster-prepare and cluster-prepare-wait) was done in https://github.com/IBM/the-mesh-for-data/pull/61 by @froesef. It allows installing the integration test stuff while the third parties are being installed.